### ISSUE TITLE
Un dossier par carte électronique

### ISSUE THREAD

**Issue by Vayel** [Poztit/pyDuino#21]
Les fichiers `pcduino.py` et `rpi.py` sont longs. Il serait peut-être plus clair de les diviser en plusieurs fichiers et de stocker ces derniers dans des dossiers `pcduino/` et `rpi/`.

Afin de pouvoir importer les principales fonctionnalités simplement (via `pyduino.pcduino.x`), il suffira de les importer dans le fichier `__init__.py`.

**Comment by fillien** [comment]
Non, je pense pas que ce soit une bonne idée. De toutes façon il va faloir ranger Uart() dans lib
